### PR TITLE
Proofreading

### PR DESCRIPTION
**Pull request by agitter**
I'm reading through end-to-end and will push any corrections I find.  I'll finish therapeutics and discussion on Wednesday.

### CODE REVIEW

**Review by rando2 — APPROVED** [review]
Thanks for catching all of these @agitter!

Just realized I shouldn't have approved, got a little overenthusiastic with all the new PRs this morning!

**Comment by agitter** [comment]
I finished my proofreading.  I added a reference to explain the concept of p-hacking, which I thought may not be common knowledge, and a vaccine review to support updated statistics (#641).

One overall concern is that the nutraceuticals section feels quite speculative given how much space is devoted to it.  These sections are well-written and referenced.  However, they mostly describe only general indications of nutraceuticals' effects on inflammation or various other viral infections and conclude we don't have much evidence about efficacy for COVID-19.

> Nutritional strategies and the use of nutraceuticals will also undoubtedly play a role in the treatment of hospitalized patients, as malnutrition is a risk to COVID-19 patients [561].

In this sentence, I think it is too strong to say nutraceuticals will undoubtedly play a role.

We also have the general challenge of having some sections that were written much earlier than others.  This particular line stood out given [Pfizer's recent press release](https://www.statnews.com/2020/11/09/covid-19-vaccine-from-pfizer-and-biontech-is-strongly-effective-early-data-from-large-trial-indicate/):

> mRNA-1273 is the only RNA vaccine for SARS-CoV-2 for which preliminary results are available.

Finally, we refer to Appendices B-D.  Are those going to be added for this submission?

File: content/20.treatments.md [hunk 1]
```diff
@@ -953,29 +953,3 @@ At present, a number of promising avenues continue to be explored.
 Potential therapeutics currently being studied target the SARS-CoV-2 or modify the host environment through many different mechanisms.
 Here, the relationship between the virus and several therapeutics described above are visualized.
 ](images/N001-LifeCyclePlusDrugs.png){#fig:therapeutics secno=1}
-
-##### Trained Immunity
```
Comment (rando2) [inline content/20.treatments.md:957]: @agitter is this section proposed to be deleted? It used to be in a different part of the manuscript so I'm not sure if that's why it's showing up weird in the diff.

**Review by rando2 — APPROVED** [review]
I just have one question about some text that looks like it might be deleted, but really helpful catches!

File: content/20.treatments.md (hunk 1 above)
Comment (agitter) [reply content/20.treatments.md:957]: Yes, those paragraphs were duplicated.  I left them in Section 5.2.4.3 and deleted them in Section 5.3.0.1.  I'm pretty sure the text is the same in both places but didn't do a diff.

**Comment by rando2** [comment]
Hi Tony,
Thanks for the detailed thoughts on this. 
- I agree and would like to see some amping up of the pharmaceuticals in the next version to make the distribution a little more even, since right now we cover a lot of different nutraceuticals and not so many pharmaceuticals. Ronan has been super active and is the resident nutraceutical expert, so that section is extremely up to date, whereas there hasn't really been one person driving the addition of new pharmaceuticals. I'm not totally sure what the best way to handle this unevenness is except to keep trying to recruit more pharmacology people (sadly I am not one, despite working in the pharmacology department).
-  I am definitely for toning down that nutrition/nutraceuticals sentence!
- We should probably change that mRNA sentence as well! Drat, I was hoping to not have to add the Pfizer press release.
- I don't remember what the appendices are supposed to be so will dig into that!

**Comment by agitter** [comment]
> I agree and would like to see some amping up of the pharmaceuticals in the next version

Maybe we defer the pharmaceuticals/nutraceuticals balance to the next version?  I do find it to be skewed heavily toward nutraceuticals, but I don't see a easy way to adjust this on our submission timeline.

> I am definitely for toning down that nutrition/nutraceuticals sentence!

I'll do this here

> We should probably change that mRNA sentence as well! Drat, I was hoping to not have to add the Pfizer press release.

I'll adjust or comment out that sentence and open an issue.

**Comment by rando2** [comment]
Thanks @agitter! I agree about deferring the balance to the next version, it took so long to shape up the existing text that I am very hesitant to try to start adding anything new. Luckily we have #605 and hoping getting this out there will help with recruiting people who are excited about new drug research!

**Comment by agitter** [comment]
I pushed a few more changes in response to the above discussion.  I'm using August 2020 to timestamp that paragraph based on #592 that added the text.

I made #713 to remind us about mRNA vaccines.  We may want to wait for a paper to add the Pfizer result.

**Comment by RLordan** [comment]
> I finished my proofreading. I added a reference to explain the concept of p-hacking, which I thought may not be common knowledge, and a vaccine review to support updated statistics (#641).
> 
> One overall concern is that the nutraceuticals section feels quite speculative given how much space is devoted to it. These sections are well-written and referenced. However, they mostly describe only general indications of nutraceuticals' effects on inflammation or various other viral infections and conclude we don't have much evidence about efficacy for COVID-19.
> 
> > Nutritional strategies and the use of nutraceuticals will also undoubtedly play a role in the treatment of hospitalized patients, as malnutrition is a risk to COVID-19 patients [561].
> 
> In this sentence, I think it is too strong to say nutraceuticals will undoubtedly play a role.
> 
> We also have the general challenge of having some sections that were written much earlier than others. This particular line stood out given [Pfizer's recent press release](https://www.statnews.com/2020/11/09/covid-19-vaccine-from-pfizer-and-biontech-is-strongly-effective-early-data-from-large-trial-indicate/):
> 
> > mRNA-1273 is the only RNA vaccine for SARS-CoV-2 for which preliminary results are available.
> 
> Finally, we refer to Appendices B-D. Are those going to be added for this submission?

Hey @agitter @rando2 I can probably shed some light on the nutraceutical issue. 

Firstly, please feel free to alter the language or content if it feels like it is too strong or indeed too speculative. I purposely wrote it speculatively as there is a lot of papers right now pushing all sorts of claims that have little basis so I thought it would be safer to be speculative on this topic. Despite that, this sector is receiving a lot of attention right now and clinical trials are underway. To put some of it into context:  Dietary supplements sales reached a revenue of $345 million in the entirety of 2019, whereas sales reached $435 million during the six-week period of the outbreak that ended on April 5 of 2020, which grew by another $151 million by May 17 2020. There is a lot of interest there. In my opinion it is unlikely that many of them do anything other than support normal immune function, but vitamin D is receiving a lot of attention and seems to be the most promising for potential prophylactic and therapeutic value. If I haven't captured those ideas, please feel free to highlight.  

On that quote in particular, that must have been a blip! Go ahead and delete nutraceuticals from the line as only basic and parenteral nutritional strategies are currently being used in hospital care right now, which is mainly occurring in Italy and other European countries.

**Comment by RLordan** [comment]
On the note of the balance, which I missed, one way to reduce the content quickly without too much disruption to the piece is to get rid the probiotics section for now? There is a lot of interest, but little evidence if you want to reduce the content more? @rando2 @agitter Happy to do it if you like?

**Comment by rando2** [comment]
Thank you @RLordan! And please don't take it negatively that you have been so thorough compared to other aspects of the paper, it's extremely impressive!!! I never even got around to adding in the part that would have been relevant to my expertise (interspecies transmission/susceptibility in a phylogenetic context -- weirdly carnivore comparative genomics are actually relevant to this paper somehow). We just eventually might want to figure out a way to apply some of the same thoroughness to other parts. I have learned so much about what has an has not been shown about nutraceuticals from working with you!

**Comment by RLordan** [comment]
> Thank you @RLordan! And please don't take it negatively that you have been so thorough compared to other aspects of the paper, it's extremely impressive!!! I never even got around to adding in the part that would have been relevant to my expertise (interspecies transmission/susceptibility in a phylogenetic context -- weirdly carnivore comparative genomics are actually relevant to this paper somehow). We just eventually might want to figure out a way to apply some of the same thoroughness to other parts. I have learned so much about what has an has not been shown about nutraceuticals from working with you!
> 
> As for probiotics, I kind of think the reviewers will tell us if they are unhappy with the balance and we can comment out or add new material in as needed? @cgreene I don't know if you have thoughts as well.

Don't worry, not taking it negatively. I, like all of you, want the best paper possible. Have a think about it, if the balance is very off let me know, I can reduce the content, no problem at all.

**Comment by rando2** [comment]
When I had previously talked to @cgreene about the number of drugs vs nutraceuticals that we have in this version, I came away with the impression that it's pretty typical for these crowd-sourced reviews to grow a bit unevenly depending on who is currently active on the project, but that they typically even out over time. So I am personally not super worried, but definitely open to suggestions as well!

And personally as someone who hears a lot of COVID-19 chat from non-scientists, I am very happy to have a source on the actual research for these popular topics! "Socially distance and make sure you're eating a balanced diet with fish and vitamin D" is a great canned response to have!

**Comment by cgreene** [comment]
This is very interesting for perspective:

> Despite that, this sector is receiving a lot of attention right now and clinical trials are underway. To put some of it into context: Dietary supplements sales reached a revenue of $345 million in the entirety of 2019, whereas sales reached $435 million during the six-week period of the outbreak that ended on April 5 of 2020, which grew by another $151 million by May 17 2020. There is a lot of interest there

Do we have this in the manuscript anywhere for context as to why we might want to be extensive?

**Comment by rando2** [comment]
Great idea @cgreene! Maybe after line 432 in Therapeutics? 

> [431] Nutraceuticals are classified as supplements with health benefits beyond their basic nutritional value [@doi:10.1080/10408390902841529; @doi:10.1038/nrcardio.2016.103].
> [432] The key difference between a dietary supplement and a nutraceutical is that nutraceuticals should not only supplement the diet, but also aid in the prophylaxis and/or treatment of a disorder or disease [@doi:10.1208/ps050325].
> [433] Unlike pharmaceuticals, nutraceuticals do not entirely fall under the responsibility of the FDA, but they are monitored as dietary supplements according to the Dietary Supplement, Health and Education Act 1994 (DSHEA) [@url:https://ods.od.nih.gov/About/DSHEA_Wording.aspx] and the Food and Drug Administration Modernization Act 1997 [@url:https://www.fda.gov/regulatory-information/selected-amendments-fdc-act/food-and-drug-administration-modernization-act-fdama-1997].

**Comment by RLordan** [comment]
> Great idea @cgreene! Maybe after line 432 in Therapeutics?
> 
> > [431] Nutraceuticals are classified as supplements with health benefits beyond their basic nutritional value [@doi:10.1080/10408390902841529; @doi:10.1038/nrcardio.2016.103].
> > [432] The key difference between a dietary supplement and a nutraceutical is that nutraceuticals should not only supplement the diet, but also aid in the prophylaxis and/or treatment of a disorder or disease [@doi:10.1208/ps050325].
> > [433] Unlike pharmaceuticals, nutraceuticals do not entirely fall under the responsibility of the FDA, but they are monitored as dietary supplements according to the Dietary Supplement, Health and Education Act 1994 (DSHEA) [@url:https://ods.od.nih.gov/About/DSHEA_Wording.aspx] and the Food and Drug Administration Modernization Act 1997 [@url:https://www.fda.gov/regulatory-information/selected-amendments-fdc-act/food-and-drug-administration-modernization-act-fdama-1997].

True! Hard to keep up to date with all the new aspects emerging. This info on sales came out in June, I only saw it last week! If you want to pop that sentence in, go ahead, here are the references for it: 

https://www.nutritionaloutlook.com/view/peak-dietary-supplement-sales-leveling-during-covid-19-pandemic-growth-still-remains-strong

&

https://doi.org/10.1016/j.tifs.2020.09.001

**Comment by agitter** [comment]
Adding this bit of extra motivation for the section could be a nice short-term solution.  Should we defer that for a follow up pull request from @RLordan and work on merging this one?  I touched a lot of files, which risks merge conflicts if it's open too long.

**Comment by cgreene** [comment]
Agree new ticket & merge this one before we wait too long

**Comment by RLordan** [comment]
> Adding this bit of extra motivation for the section could be a nice short-term solution. Should we defer that for a follow up pull request from @RLordan and work on merging this one? I touched a lot of files, which risks merge conflicts if it's open too long.

Perfect, should I go ahead and set up a pull request for that addition? @cgreene @rando2

**Comment by rando2** [comment]
That would be perfect @RLordan! I'll go ahead and merge this now.

**Pull request merged**